frontend/apps/filemanager/filemanagerhistory.lua

```diff
@@ -237,19 +236,13 @@ function FileManagerHistory.getMenuInstance()
 end
 
 function FileManagerHistory:onShowHist(search_info)
-    self.hist_menu = Menu:new{
-        ui = self.ui,
-        covers_fullscreen = true, -- hint for UIManager:_repaint()
-        is_borderless = true,
-        is_popout = false,
+    self.hist_menu = BookList:new{
         title = self.hist_menu_title,
-        -- item and book cover thumbnail dimensions in Mosaic and Detailed list display modes
-        -- must be equal in File manager, History and Collection windows to avoid image scaling
-        title_bar_fm_style = true,
```

Comment:
This informative comment is no longer relevant ? Because all of these now inherit from BookList so it will always be ensure ? Right ? (yes/no is enough :))

Reply:
Yes